### DISCUSSION TITLE
OUSTER | COMMUNITY FORUM

### DISCUSSION THREAD

**Discussion by Samahu** [ouster-lidar/ouster_example#592] [Announcements]
For additional resources about ouster sensors and the different software the company offers please visit the newly launched ouster forum: https://community.ouster.com there you can post questions and get answers from the community and follow topics of interest like ouster-studio, ouster-gemini, or the ouster-sdk.

**Comment by zainkafeel** [comment]
Anywhere Example Vehicle   Speed Detection using liDar

**Reply by Samahu** [reply]
We don't have a good example for this but for object tracking from Lidar with YOLO you can take a look at: https://github.com/ouster-lidar/ouster-yolov5-demo (it is probably based of SDK 0.13 though).. this covers the first method outlined in my response here: https://community.ouster.com/t/vehicle-speed-detection/443/2
